tests/release/README.md

```diff
@@ -0,0 +1,46 @@
+# Release e2e tests
+
+Contains E2E tests related to [Release Service](https://github.com/redhat-appstudio/infra-deployments/tree/main/components/release).
+
+This document will detail the e2e tests included for the release suite.
+
+## e2e tests
+### 1. Good Path (release.go)
+   This test verifies the functionality of the release-service in watching for new release CRs then performing the necessary actions based on the design.
+
+      Steps:
+       - Create the following CRDs: Snapshot, ReleaseStrategy, ReleasePlan, EnterpriseContract, ReleasePlanAdmission, and Release.
```

Comment:
You are creating CRs of the following kinds really... creating a CRD is creating a new definition